code/game/atoms.dm

```diff
@@ -1051,7 +1051,7 @@
 /atom/proc/component_storage_contents_dump_act(obj/item/src_object, mob/user)
 	var/list/things = src_object.contents
 	var/datum/progressbar/progress = new(user, things.len, src)
-	while (do_after(user, 1 SECONDS, src, NONE, FALSE, CALLBACK(src_object.atom_storage, /datum/storage.proc/handle_mass_transfer, user, src)))
+	while (do_after(user, 1 SECONDS, src, NONE, FALSE, CALLBACK(src_object.atom_storage, /datum/storage.proc/handle_mass_transfer, user, src, TRUE)))
```

Comment:
Named parameters don't work here, so instead make a comment like this:

```suggestion
	while (do_after(user, 1 SECONDS, src, NONE, FALSE, CALLBACK(src_object.atom_storage, /datum/storage.proc/handle_mass_transfer, user, src, /* parameter_name = */ TRUE)))
```

It is currently impossible to know what the TRUE there is for.